src/app.js

```diff
@@ -1,21 +1,57 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+const [enteredSourcePath, enteredPath] = process.argv.slice(2);
+
+const resolvedPath = path.resolve(__dirname, '../', enteredPath);
+
+const resolvedSourcePath = path.resolve(__dirname, '../', enteredSourcePath);
+const fileName = enteredSourcePath.split('/').slice(-1);
+
+
+const moveFile = (sourceFilePath, destinationFilePath) => {
+  try {
+    fs.copyFileSync(sourceFilePath, destinationFilePath);
+  } catch (error) {
+    throw new Error(`An error ocurred during copying file: ${error}`);
+  }
+
+  try {
+    fs.rmSync(sourceFilePath)
+  } catch (error) {
+    throw new Error(`An error ocurred during removing the old file: ${error}`);
+  }
+
+  console.info('File successfully moved');
 }
 
-module.exports = sum;
+const getStats = (path) => {
+  try {
+    const stats = fs.statSync(path);
+    return stats;
+  } catch (_e) {
```

Comment:
e🥲

Reply:
yeah 🙄 
I didn't have time to adjust/fix the linter - it doesn't work locally, so wasn't sure that `_` allows to leave unused vars
I'll change the name